src/main/java/jalau/at18/azul/Pyramide.java

```diff
@@ -0,0 +1,19 @@
+package jalau.at18.azul;
+
+public class Pyramide {
+    private final int pyramideSize = 5;
+    private final int pyramideLimit = 4;
```

Comment:
Instead of `size` and `limit`, would it make sense to use `height` and `width` for the variable names? adds a little bit more of clarity on the purpose of these constants